src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -37,6 +37,11 @@ import VariantCountCache from "./VariantCountCache";
 
 type PageMode = 'patient' | 'sample';
 
+class OncokbDefault implements IOncoKbData {

```

Comment:
this as a class doesn't quite feel right to me.. it should really be a constant, I think.

`const ONCOKB_DEFAULT:IOncoKbData = { sampleToTumorMap: {}, indicatorMap: {} };`

remoteData returns immutable, right?